src/js-tracker.js

```diff
@@ -95,53 +108,49 @@ function TrackerAsset() {
 
         var tracker = this;
         this.HttpRequest(this.url + 'login', 'post', {'Content-Type': 'application/json' }, JSON.stringify({username: username, password: password}),
-         function (data) {
-            tracker.settings.userToken = 'Bearer ' + data.user.token;
-            if (tracker.settings.debug) {
-                console.info('AuthToken: ' + data.user.token);
-            }
-            callback(data, null);
-
-        },
-         function (data) {
-            if (tracker.settings.debug && data.responseJSON) {
-                console.log(data.responseJSON);
-            }
-            callback(data, true);
-        });
+            function (data) {
+                if (tracker.settings.debug && data.responseJSON) {
+                    console.log(data.responseJSON);
+                }
+                callback(true, data);
+            },
+            function (data) {
+                tracker.settings.userToken = 'Bearer ' + data.user.token;
+                if (tracker.settings.debug) {
+                    console.info('AuthToken: ' + data.user.token);
+                }
+                callback(null, data);
+            });
     };
 
     this.LoginBeaconing = function(accessToken, callback) {
         this.generateURL();
 
         var tracker = this;
         this.HttpRequest(this.url + 'login/beaconing', 'post', {'Content-Type': 'application/json' }, JSON.stringify({accessToken: accessToken}),
```

Comment:
Open an issue to track this URLs down. Any reference to beaconing / rage should be removed eventually. They are just dead code not required anymore.

Reply:
issue opened